src/nunjucks/moduk/components/breadcrumbs/__examples__/default.njk

```diff
@@ -0,0 +1,18 @@
+{%- from "moduk/components/breadcrumbs/macro.njk" import modukBreadcrumbs -%}
+
+{{ modukBreadcrumbs({
+  items: [
+    {
+      text: "Home",
+      href: "#"
+    },
+    {
+      text: "Passports, travel and living abroad",
+      href: "#"
+    },
+    {
+      text: "Travel abroad",
+      href: "#"
+    }
+  ]
+}) }}
```

Comment:
Shoud I add a new line here, should this be a lint rule

Reply:
Added a comment about adding an `.editorconfig` – hopefully that will stop it happening. Maybe we can add https://www.djlint.com/docs/languages/nunjucks/ too.